lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -1011,4 +1026,25 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
 
   defp parse_hana_cluster_health(%{sr_health_state: _, secondary_sync_state: _}),
     do: Health.critical()
+
+  defp parse_hana_scenario(crmmon, ClusterType.hana_scale_up()) do
```

Comment:
I don't get all of this. I thought we agreed that we were just checking if `additional_sids` have some element to decide if it is cost or performance optimized.
```
defp parse_hana_scenario([]), do: HanaScenario.performance_optimized()
defp parse_hana_scenario(_), do: HanaScenario.cost_optimized()
```
We don't need to check the resources again, we already did